FiveCalls/FiveCalls/AppState.swift

```diff
@@ -91,4 +92,13 @@ extension AppState {
 
         return contactIDs.contains(contactID)
     }
+    
+    var needsIssueRefresh: Bool {
+        if issueFetchTime == nil {
```

Comment:
A little less prone to error if we ever make changes to do something like:
```
guard let issueFetchTime else {
   return true
}

if issueFetchTime < Date().addingTimeInterval(-1 * 60) {
    return true
} else {
    return false
}

Reply:
Good call, much easier to think about this way